bin/pegjs

```diff
@@ -183,8 +183,8 @@ while (args.length > 0 && isOption(args[0])) {
       if (args.length === 0) {
         abort("Missing parameter of the --format option.");
       }
-      if (args[0] !== "amd" && args[0] !== "commonjs" && args[0] !== "globals" && args[0] !== "umd") {
-        abort("Module format must be one of \"amd\", \"commonjs\", \"globals\", and \"umd\".");
+      if (args[0] !== "amd" && args[0] !== "commonjs" && args[0] !== "esm" && args[0] !== "globals" && args[0] !== "umd") {
+        abort("Module format must be one of \"amd\", \"commonjs\", \"esm\", \"globals\", and \"umd\".");
```

Comment:
I think it would be slightly better to identify the format as `es2015` — it is more readily understandable. But if most common tools that work with module formats use `esm`, then it would be probably better to keep it (I’d like to see examples of such tools then).

The condition also got a bit too long — maybe extract the list of formats into a const  named e.g. `MODULE_FORMATS` and rewrite the test as `MODULE_FORMATS.indexOf(args[0]) === -1`? Up to you. Best done as a separate commit before the addition itself.